rj_gameplay/classifier/fumble_classifier.py

```diff
@@ -0,0 +1,123 @@
+import stp.rc as rc
+import rj_gameplay.gameplay_node as gameplay_node
+import rclpy
+
+
+def fumble_classifier(world_state: rc.WorldState, possess) -> bool:
+	if possess is None:
+		return True
+	else: 
+		return False
+
+"""
+get_distance returns the distance between the ball and the kicker
+num: kicker's id
+"""
+def get_distance(world_state: rc.WorldState, kicker) -> float:
+	if world_state is not None:
+		robot = world_state.our_robots[kicker]
+		pose_x = robot.pose[0]
+		pose_y = robot.pose[1]
+		ball_pos_x = world_state.ball.pos[0]
+		ball_pos_y = world_state.ball.pos[1]
+		distance = ((ball_pos_x-pose_x)**2 + (ball_pos_y-pose_y)**2)**0.5
+		return distance
+
+"""
+get_velocity_angle returns the ratio of y component of the velocity vector to x component
+"""
+
+def get_velocity_angle(world_state: rc.WorldState) -> float:
```

Comment:
Based on how it's used, I would suggest making this an actual angle (see: `math.atan2` or `np.atan2`), and then dealing with angle wrap accordingly when comparing two angles. If it absolutely needs to be a ratio like this, please change the name.